
Header header
uint8 client_command_mode
float64[] commanded_joint_position
uint8 connection_quality
uint8 control_mode
uint8 drive_state
float64[] ipo_joint_position
uint8 operation_mode
uint8 overlay_type
uint8 safety_state
float64 sample_time
uint8 session_state
time cabinet_timestamp
float64 tracking_performance
